src/Compilers/CSharp/Test/Semantic/Semantics/RecordStructTests.cs

```diff
@@ -10568,5 +10568,38 @@ public void BaseArguments_Speculation()
 
             Assert.False(model.TryGetSpeculativeSemanticModel(baseWithoutParens.SpanStart + 2, speculativeBase, out _));
         }
+
+        [Fact, WorkItem(54413, "https://github.com/dotnet/roslyn/issues/54413")]
+        public void Repro54413()
+        {
+            var src = @"
+record struct Value(string Text)
+{
+    private Value(int X) { } // 1
+    private Value(Value original) { } // 2
```

Comment:
Do we have a passing test with a `record struct` with a constructor that has the signature of a copy constructor?